5-12-2018/monika/monika-sample/config/routes.rb

```diff
@@ -0,0 +1,11 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root 'users#index'
```

Comment:
No need for specifying separate routes for only CRUD, use resources or even namespace if you really want to control all routes by yourself